wgengine/magicsock/peermtu.go

```diff
@@ -110,3 +115,7 @@ func (c *Conn) UpdatePMTUD() {
 	c.peerMTUEnabled.Store(newStatus)
 	c.resetEndpointStates()
 }
+
+func isPMTUError(err error) bool {
```

Comment:
This should probably move files and be named in a way that it applies more generically, e.g. `isEMSGSizeErr()`.